src/basicfuns.jl

```diff
@@ -149,6 +149,31 @@ end
 """
 $(SIGNATURES)
 
+Return `log(abs(tanh(x)))`, carefully evaluated without intermediate calculation of `tanh(x)`.
+
+The implementation ensures `logabstanh(-x) = logabstanh(x)`.
+"""
+function logabstanh(x::Real)
+    return log1p(-2/((exp(2abs(x))+1)))
+end
+function logabstanh(x::Float32)
+    abs_x = abs(x)
+    if abs_x < 0.0625f0
+        return log(abs_x) - x*x*(1f0/3)
```

Comment:
Any reason to not just use
```suggestion
        return log(abs_x) - x^2/3
```
?